target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -552,6 +552,21 @@ define Device/tplink_tl-wr842n-v3
 endef
 TARGET_DEVICES += tplink_tl-wr842n-v3
 
+define Device/tplink_tl-wr902ac-v1
+  $(Device/tplink-safeloader)
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := TP-Link
```

Comment:
That might be invoked by tplink-safeloader

Reply:
Indeed it is, removed.